core/src/main/scala/org/locationtech/rasterframes/expressions/transformers/ReprojectGeometry.scala

```diff
@@ -82,8 +82,11 @@ case class ReprojectGeometry(geometry: Expression, srcCRS: Expression, dstCRS: E
       // Optimized pass-through case.
       case (s: LazyCRS, r: LazyCRS) if s.encoded == r.encoded => geometry.eval(input)
       case _ =>
-        val geom = JTSTypes.GeometryTypeInstance.deserialize(geometry.eval(input))
-        JTSTypes.GeometryTypeInstance.serialize(reproject(geom, src, dst))
+        if (geometry.eval(input) != null) {
+          val geom = JTSTypes.GeometryTypeInstance.deserialize(geometry.eval(input))
+          JTSTypes.GeometryTypeInstance.serialize(reproject(geom, src, dst))
+        }
+        null
```

Comment:
Why do you always evaluate to null here... should this be in an `else`?

Reply:
Oops. you're right.
Fix that